### PR TITLE
CLOG-154 Edit Comments button now gets post ID correctly

### PR DESCRIPTION
**Pull request by washingtonj3**
Edit Comments button now gets post ID correctly
New and edited comments now redirect to the post instead of the list of posts

### CODE REVIEW

**Comment by adrianfish** [comment]
Tell me when you're done :)

File: tool/src/webapp/js/clog.js [hunk 1]
```diff
@@ -409,7 +409,7 @@ clog.switchState = function (state,arg) {
                 });
             };
 
-        clog.utils.findPost(arg.postId, commentCallback);
+        clog.utils.findPost(arg.postId, commentCallback, arg.fromSamepage);
 	} else if ('permissions' === state) {
```
Comment (adrianfish) [inline tool/src/webapp/js/clog.js]: Pedantry time. Can you adjust the indentation? Mega small, but still :)

File: tool/src/webapp/js/clog.js [hunk 2]
```diff
@@ -277,7 +281,7 @@ clog.switchState = function (state,arg) {
             };
 
 		if (arg && arg.postId) {
-			clog.utils.findPost(arg.postId, postCallback);
+			clog.utils.findPost(arg.postId, postCallback, arg.fromSamepage);
         }
 	} else if ('createPost' === state) {
```
Comment (adrianfish) [inline tool/src/webapp/js/clog.js]: More indentation.

File: tool/src/webapp/js/clog_utils.js [hunk 3]
```diff
@@ -377,12 +378,11 @@ clog.utils = {
...
     },
-    findPost: function (postId, callback) {
-        
-        if (!clog.currentPosts) {
+    findPost: function (postId, callback, fromSamepage) {
 
-            jQuery.ajax( {
-                url: "/direct/clog-post/" + postId + ".json",
+        if (!clog.currentPosts || fromSamepage) {
+            jQuery.ajax({
```
Comment (adrianfish) [inline tool/src/webapp/js/clog_utils.js]: Can you indent all this in the same. Cheers.

Comment (washingtonj3) [inline tool/src/webapp/js/clog_utils.js]: The indentations have been fixed.
I could not find anything that needed to be changed with the indentations on lines 380-384

**Comment by ottenhoff** [comment]
wdyt @adrianfish ?

**Pull request merged**